spring-cloud-dataflow-docs/src/main/asciidoc/api-guide.adoc

```diff
@@ -2643,3 +2644,118 @@ include::{snippets}/runtime-apps-documentation/list-all-apps/curl-request.adoc[]
 
 include::{snippets}/runtime-apps-documentation/list-all-apps/http-response.adoc[]
 
+
+[[api-guide-resources-stream-logs]]
+=== Stream Logs
+
+It is possible to get the application logs of the stream for the entire stream or a specific application inside the stream.
+The following topics provide more detail:
+
+* <<api-guide-resources-stream-logs-by-stream-name>>
+* <<api-guide-resources-stream-logs-by-app-name>>
+
+
+
+[[api-guide-resources-stream-logs-by-stream-name]]
+==== Get the applications' logs by the stream name
+
+To retrieve the logs of all the application of stream by the given stream name, query the `/streams/logs/<streamName>` endpoint by using `GET`.
```

Comment:
Use the HTTP `GET` method with the `/streams/logs/<streamName>` REST endpoint to retrieve all the application logs for the given stream name.